SPECS/xcp-python-libs.spec

```diff
@@ -36,6 +36,7 @@ Summary: Common XCP-ng Python classes for Python3
 # See https://github.com/xenserver/python-libs/blob/master/pyproject.toml:
 Requires: python3-six
 Requires: biosdevname
+Requires: xenserver-release >= 8.3.60-5
```

Comment:
Anyone checked that we do provide xenserver-release in xcp-ng-release? :sweat_smile:

Reply:
I don't think we do.

Reply:
shouldn't we remove this requirements ?

Reply:
Or adapt it. Or "provide" (RPM `Provides` tag) xenserver-release in xcp-ng-release.

Reply:
I have a looked and in fact xenserver-release package is our xcp-ng-release. It provides some constants used in python scripts. I compared the two versions (xenserver-release-8.3.60-5 and xcp-ng-release-8.3.0-28) and they provide  the same information. So if we are following the same versioning we can probably replace it by  `xcp-ng-release-XXX`.  I don't know if it is the right way to go and what version we should use. Also it looks like we can just drop... Because I also compared xenserver-release-8.3.50-5 vs xenserver-release-8.3.60-5 and I don't see new variables. So I don't understand this requirement.

Reply:
I removed the dependency

Reply:
So I had forgotten about this already, went back into the git history, preparing the forum announcement, and had a hard time understanding the diff, with a changelog that days it's adding a dependency and actually we don't. 

So to be noted for the future: we should have mentioned the fact that we removed the offending line in the commit message, or created a commit just for it.

Or we should start generalizing the approach where stuff from XS we don't like we don't delete, but rather enclose with conditionals.